src/System Application/App/Company Initialize/CompanyInitialize.Codeunit.al

```diff
@@ -0,0 +1,19 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Environment.Configuration;
+
+codeunit 900 "Company Initialize"
```

Comment:
This should be moved to the platform system events. Whenever a new company is created they should trigger the initialize company event.

Reply:
This is a nice opportunity to improve the flows.